README.md

```diff
@@ -53,6 +53,8 @@ indirect business models around the free open-source software platform
 provided by RIOT, e.g. it is possible to link closed-source code with the
 LGPL code.
 
+[Ariel OS](https://github.com/ariel-os/ariel-os) is a offsping of RIOT rewritten in Rust following RIOTs targets.
```

Comment:
[Ariel OS](https://github.com/ariel-os/ariel-os) is an offspring of RIOT rewritten in Rust following RIOTs targets.

(grammar)

Reply:
Also: "RIOTs" is plural, I think you mean posessive "RIOT's".

Reply:
s/targets/vision? I mean, we even have a doc page for that: https://doc.riot-os.org/vision.html